components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/validators/AbstractRulesValidator.java

```diff
@@ -93,6 +96,21 @@ public boolean validateProps(ValidationContext context) throws InputValidationMg
                     String.format(ERROR_DEFAULT_MIN_MAX_MISMATCH.getDescription(), this.getClass().getSimpleName(),
                             properties.get(MIN_LENGTH), properties.get(MAX_LENGTH)));
         }
+
+        // Validate the max length for the password field.
+        if (PASSWORD.equals(context.getField())) {
+            int maxPasswordValue = Integer.parseInt(IdentityUtil.getProperty(MAX_PASSWORD_ALLOWED_LENGTH));
+            if (properties.get(MAX_LENGTH) != null &&
+                    Integer.parseInt(properties.get(MAX_LENGTH)) > maxPasswordValue) {
+                if (log.isDebugEnabled()) {
+                    log.error(String.format("The property %s should be less than or equal to %s for the tenant %s.",
```

Comment:
Let's change this to debug log

Reply:
Fixed by https://github.com/wso2/carbon-identity-framework/pull/6182/commits/e58da0b864bb4f189b914265dfe4e5ad532f676a